module/VuFind/src/VuFind/RecordDriver/DefaultRecord.php

```diff
@@ -282,6 +282,28 @@ public function getCleanISBN()
         return $isbn13;
     }
 
+    /**
+     * Return all valid ISBNs found in the record.
+     *
+     * @return array
+     */
+    public function getCleanISBNs(): array
```

Comment:
Would it make sense to include a flag or option array to potentially allow the caller to filter for ISBN-10 only, or to normalize to ISBN-13? If you don't think this would be useful, I don't insist on it... just thinking about potentially useful features while we're designing something new.

Reply:
I tried to add the flags array. Is that what you have in mind?